types/foundry/client/core/text-editor.d.ts

```diff
@@ -140,7 +140,7 @@ declare global {
          * Handle left-mouse clicks on an inline roll, dispatching the formula or displaying the tooltip
          * @param event The initiating click event
          */
-        static _onClickInlineRoll(event: MouseEvent): Promise<ChatMessage>;
```

Comment:
scanning the source, it looks like this should be `Promise<ChatMessage | void>`

Reply:
In terms of what it does maybe, but given this is just a listener the return type doesnt actually matter. Foundry only returns chat message so thst its async i imagine.

Reply:
It's async merely by having an `async` modifier (which isn't seen in the type)